wiki/Glossary/en.md

```diff
@@ -486,21 +487,21 @@ Upon the creation of the marathon beatmap, one should name their beatmap version
 
 ### MAT
 
-**As of the 23. April 2013, all members of the MAT at that time were promoted as full members of the [BAT](/wiki/BAT), with the MAT team decommissioned.**
+**As of the 2013-04-23, all members of the MAT at that time were promoted as full members of the [BAT](/wiki/BAT), with the MAT team decommissioned.**
```

Comment:
Not sure why this was written this way, it can probably be merged into the paragraph below.